addon/mixins/sizing-mixin.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @module ticketfly-fields
+ */
+import Ember from 'ember';
+const { Mixin, computed, get } = Ember;
+
+/**
+  @class SizingMixin
+  @extends Ember.Mixin
+ */
+export default Mixin.create({
+  sizing: computed('sizeStyle','utilityName', {
+    get() {
+      const sizeStyle = get(this, 'sizeStyle');
+      if (sizeStyle) {
+        const utilityName = get(this, 'utilityName');
+        return `u-${utilityName}-${sizeStyle}`;
```

Comment:
Should this also be more BEM-like? 

Also, does anyone have any feelings about the dynamically generated class names? 

My feeling is that the class names that we have should be a bit more static as, in a way, the class names are becoming part of the public API. With this approach, the consumer of the component could put anything in for `sizeStyle` and it'd generate a corresponding class name wether or not we had a CSS class definition that could support that style. 

This might be okay; but it might also be nice to have some guardrails for developers using this so it becomes self-documenting in a way: For example, if you provided a non-valid option, we could write a warning to the console and say something like `TF-Input Warning: 'zomgHuge' is not a valid 'sizeStyle'. The valid options are 'small', 'medium', and 'large'.`